public/app/features/alerting/partials/notification_edit.html

```diff
@@ -32,6 +32,13 @@ <h3 class="page-sub-heading" ng-show="ctrl.isNew">New Notification Channel</h3>
           checked="ctrl.model.settings.uploadImage"
           tooltip="Captures an image and include it in the notification">
       </gf-form-switch>
+      <gf-form-switch
+          class="gf-form"
+          label="Disable OK Alert"
```

Comment:
The UI Option is named differently form the domain property. Should be aligned. 
Not sure what the best name for this option is (or if we should opt for a more generic filtering solution)